webview/cocoa.py

```diff
@@ -57,6 +57,16 @@ class BrowserDelegate(AppKit.NSObject):
         def webView_contextMenuItemsForElement_defaultMenuItems_(self, webview, element, defaultMenuItems):
             return nil
 
+        # Display a JavaScript alert panel containing the specified message
+        def webView_runJavaScriptAlertPanelWithMessage_initiatedByFrame_\
+                (self, webview, message, frame):
+                    AppKit.NSRunningApplication.currentApplication().activateWithOptions_(AppKit.NSApplicationActivateIgnoringOtherApps)
+                    alert = AppKit.NSAlert.alloc().init()
+
+                    alert.setMessageText_(localization["cocoa.alert"])
```

Comment:
You are right. In fact I wrote it this way after some consideration. It didn't feel right without a title because unlike the `window.alert` displayed by Safari, the native `NSAlert` we use _does_ usually have a title:

![screen shot 2017-04-09 at 11 45 52 pm](https://cloud.githubusercontent.com/assets/18247990/24839965/b103e54c-1d81-11e7-9965-653ee2dacaa6.png)

_Without title (not pretty)_
![screen shot 2017-04-09 at 11 49 53 pm](https://cloud.githubusercontent.com/assets/18247990/24839894/34054528-1d80-11e7-8275-a9e670ab2288.png)

So I don't advocate making the title empty. Another option is to make the `message` itself the title (and omit the `informativeText` property), but that looks ugly and non-standard:

_This would be uglier if the message is something long_
![screen shot 2017-04-09 at 11 50 41 pm](https://cloud.githubusercontent.com/assets/18247990/24839893/33ff7030-1d80-11e7-9c1e-5baa7fedc7df.png)

I just figured out it's better to _omit_ it as you said, because macOS shows `Alert` as the default title. So yes, if that fixes it, I'll make the changes.